### DISCUSSION TITLE
How do I properly set up the ip address in a docker container in UnRaid?

### DISCUSSION THREAD

**Discussion by HamGoblin** [Impostor/Impostor#298] [Q&A]
Sorry I'm very new to all this, hoping I can get a bit of help here. 

I have my docker container set up and mapped in UnRaid. However, I'm not entirely sure what to set my IP too in my config.json file. I'm using "docker inspect Imposter" to find the ip of the container and assigning that, but when I try to join the server, I get "Reliable packet 1 (size=18) was not ack'd after 7522ms (9 resends)"

Thanks for any and all help.

**Comment by donalj** [comment]
Have you forwarded ports on your router to your Unraid server e.g.
![image](https://user-images.githubusercontent.com/15871751/110123809-4aa9dd00-7db9-11eb-8835-43aab8605fa2.png)

Unraid docker can be set up as follows:
![image](https://user-images.githubusercontent.com/15871751/110123883-5d241680-7db9-11eb-87e6-150f227fe0ae.png)

Then you can set your `config.json` as this:
```
{
    "Server": { 
        "PublicIp": "<PUBLIC_IP>",  # this can be found at https://www.hashemian.com/whoami/
        "PublicPort": 22023,
        "ListenIp": "0.0.0.0",
        "ListenPort": 22023
    }
}
```
